docs/aws/README.md

```diff
@@ -11,36 +13,54 @@ export AWS_DEFAULT_OUTPUT=json
 ```
 
 ### Create a Cluster
-Note: If you already have a cluster with version 1.19 or above, you may need to manually tag your subnets for Karpenter to work [detailed here](https://github.com/awslabs/karpenter/issues/404#issuecomment-845283904).
 
-If your cluster version is 1.18 or below, you can skip this step.
+Create an EKS cluster
 ```bash
 eksctl create cluster \
 --name ${CLUSTER_NAME} \
---version 1.18 \
---region ${AWS_DEFAULT_REGION} \
 --node-type m5.large \
 --nodes 1 \
```

Comment:
@ellistarn Is there a reason we don't make this cluster with `--fargate`? If karpenter is going to add all of our instances to the cluster what's the point of having a node group at all?

Reply:
This is an interesting idea. I kinda of like this idea since it creates a more "managed" karpenter experience. However, you do need *some* nodes to run core-dns, so we'd need to run those in fargate, too.

Reply:
using the `--fargate` option automatically schedules `kube-system` and `default` workloads onto fargate. So coredns will run. We can either add a config file to include the `karpenter` namespace or schedule karpenter into `default`. The example workloads would need to be schedule into a different namespace and karpenter can handle adding nodes for that.

Reply:
I'm pretty sure I can get this to work with fargate using an eksctl config file. I'm going to make that a separate PR because it'll need to move around a few commands and I don't want this PR to get complicate.